api/app/Models/CandidateSearchRequest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * Class Department
+ *
+ * @property int $id
+ * @property int $full_name
+ * @property array $email
+ * @property array $department_id
+ * @property array $job_title
+ * @property array $additional_comments
+ * @property array $summary_of_filters
```

Comment:
`$summary_of_filters` isn't referenced anywhere else, can it be removed?

Reply:
Yup that should of been removed